.travis.yml

```diff
@@ -38,6 +40,8 @@ install:
       echo "Installing a fresh version of Miniconda."
       MINICONDA_URL="https://repo.continuum.io/miniconda"
       MINICONDA_FILE="Miniconda3-latest-MacOSX-x86_64.sh"
+      # temporarily use an older miniconda so that we can get a build out
+      MINICONDA_FILE="Miniconda3-4.5.4-MacOSX-x86_64.sh"
```

Comment:
This shouldn't be needed any more now that  osx-64/conda-forge-ci-setup-2.1.1-py37_0.tar.bz2 has been published on anaconda.org .

Reply:
we'll change it in 2.1.2.  Mostly harmless for now

Reply:
and this branch can't get any more commits really in any case since they all make releases

Reply:
Sure, I didn't notice this was open directly from the `branch2.0` branch.